vnext/Microsoft.ReactNative.Cxx/ReactPromise.cpp

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ReactPromise.h"
+#include "JSValueWriter.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+static const char *ErrorDefaultCode = "EUNSPECIFIED";
```

Comment:
>ErrorDefaultCode [](start = 19, length = 16)

nit: To me these seem more readbable with words reordered. E.g. `DefaultErrorCode`

Reply:
I will think about it.

---
In reply to: [356921223](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356921223) [](ancestors = 356921223)